frontend/public/models/yaml-templates.ts

```diff
@@ -693,4 +693,48 @@ spec:
     type: JenkinsPipeline
   triggers:
   - type: ConfigChange
+`).setIn([referenceForModel(k8sModels.ResourceQuotaModel), 'rq-compute'], `
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: compute-quota
+  namespace: test-ns
```

Comment:
Leave namespace off of the templates. It gets pre-filled with the current namespace.